src/contract-details/ContractDetailsCard.js

```diff
@@ -48,24 +49,25 @@ export default class ContractDetailsCard extends Component {
 						</tr>
 					</tbody>
 				</table>
-				{(validToSell && !validationError) ?
-					<button
-						className="sell-at-market-btn"
-						onClick={() => actions.sellContract(contract.contract_id, 0)}
-					>
-						Sell at Market (<NumberPlain
-							value={contract.bid_price}
-							currency={contract.currency}
-							isProfit={v => v - contract.buy_price}
-						/>)
-					</button> :
+				{(durationUnit !== 't') && (
```

Comment:
After we start having these conditionals, even if they are correct, we need to extract to a 'biz logic' function/lib somewhere + unit test them